XML/Other/ignorable-boot-errors.xml

```diff
@@ -19,29 +19,34 @@
             <keywords>display-manager.service</keywords>
             <keywords>ACPI PCC probe failed</keywords>
             <keywords>Failed to access perfctr msr</keywords>
+            <keywords>Failed to init entropy source hwrng</keywords>
+            <keywords>augenrules.*: failure 1</keywords>
+            <keywords>microcode_ctl: kernel version \".*\" failed early load check for \".*\", skipping</keywords>
+            <keywords>rngd: Failed to init entropy source 0: Hardware RNG Device</keywords>
         </failures>
         <errors>
             <keywords>TLS record write failed</keywords>
             <keywords>state ensure error</keywords>
             <keywords>got unexpected HTTP status code</keywords>
             <keywords>Error getting hardware address for "eth0": No such device</keywords>
-            <keywords>codec can't encode character</keywords>
-            <keywords>Failed to set certificate key file [gnutls error -64: Error while reading file.]</keywords>
-            <keywords>audispd: Error - /etc/audisp/plugins.d/wdgsmart-syslog.conf isn't owned by root</keywords>
+            <keywords>codec can\'t encode character</keywords>
+            <keywords>Failed to set certificate key file \[gnutls error -64: Error while reading file.\]</keywords>
+            <keywords>audispd: Error - \/etc\/audisp\/plugins.d\/wdgsmart-syslog.conf isn\'t owned by root</keywords>
             <keywords>Condition check resulted in Process error reports when automatic reporting is enabled</keywords>
             <keywords>error trying to compare the snap system key: system-key missing on disk</keywords>
-            <keywords>open /dev/vmbus/hv_fcopy failed; error: 2 No such file or directory</keywords>
+            <keywords>open \/dev\/vmbus\/hv_fcopy failed; error: 2 No such file or directory</keywords>
```

Comment:
it have some issue, since we have add it into ignorable list, but in your test result on ubuntu, it still show up.

Reply:
Thank you @LiliDeng ! It because this line hits both errors and failures. I just fixed it and remove some redundant '\'. Not these errors/failures left in Ubuntu:
```
04/29/2020 04:14:48 PM : ERROR : Found ERROR/WARNING/FAILURE messages in logs.
04/29/2020 04:14:48 PM : INFO : Errors: /var/log/syslog:716:Apr 29 02:37:57 LISAv2-OneVM-walaautodev-ZG82-20200428193544-role-0 kernel: [    2.499479] GPT: Use GNU Parted to correct GPT errors.
04/29/2020 04:14:48 PM : INFO : Errors: /var/log/syslog:746:Apr 29 02:37:57 LISAv2-OneVM-walaautodev-ZG82-20200428193544-role-0 kernel: [    2.759179] RAS: Correctable Errors collector initialized.
04/29/2020 04:14:48 PM : INFO : Errors: /tmp/dmesg:412:[    2.499479] GPT: Use GNU Parted to correct GPT errors.
04/29/2020 04:14:48 PM : INFO : Errors: /tmp/dmesg:442:[    2.759179] RAS: Correctable Errors collector initialized.
04/29/2020 04:14:48 PM : INFO : failures: /var/log/syslog:280:Apr 29 02:37:57 LISAv2-OneVM-walaautodev-ZG82-20200428193544-role-0 systemd[1]: hv-fcopy-daemon.service: Main process exited, code=exited, status=1/FAILURE
04/29/2020 04:14:48 PM : INFO : failures: /var/log/syslog:281:Apr 29 02:37:57 LISAv2-OneVM-walaautodev-ZG82-20200428193544-role-0 systemd[1]: hv-fcopy-daemon.service: Failed with result 'exit-code'.
```

Reply:
It looks like these 2 lines should be ignored right?
```
04/29/2020 04:14:48 PM : INFO : Errors: /var/log/syslog:716:Apr 29 02:37:57 LISAv2-OneVM-walaautodev-ZG82-20200428193544-role-0 kernel: [    2.499479] GPT: Use GNU Parted to correct GPT errors.
04/29/2020 04:14:48 PM : INFO : Errors: /var/log/syslog:746:Apr 29 02:37:57 LISAv2-OneVM-walaautodev-ZG82-20200428193544-role-0 kernel: [    2.759179] RAS: Correctable Errors collector initialized.
```
And about the hv-fcopy-daemon.service failures I've no idea about that.

Reply:
It looks the first one "GPT: Use ..." is known bug in the system - https://bugzilla.redhat.com/show_bug.cgi?id=996445. The second one was escalated to Canonical, https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1791193 But both do not seem to be serious. I think "ignorable"

Reply:
OK Thanks @juhlee-microsoft ! Please wait a moment and I'll add these messages into ignorable-boot-errors.xml.

Reply:
Updated. Now only these 2 failures in Ubuntu:
```
04/30/2020 02:20:10 AM : ERROR : Found ERROR/WARNING/FAILURE messages in logs.
04/30/2020 02:20:10 AM : INFO : failures: /var/log/syslog:280:Apr 29 02:37:57 LISAv2-OneVM-walaautodev-ZG82-20200428193544-role-0 systemd[1]: hv-fcopy-daemon.service: Main process exited, code=exited, status=1/FAILURE
04/30/2020 02:20:10 AM : INFO : failures: /var/log/syslog:281:Apr 29 02:37:57 LISAv2-OneVM-walaautodev-ZG82-20200428193544-role-0 systemd[1]: hv-fcopy-daemon.service: Failed with result 'exit-code'.
```